usps/main.py

```diff
@@ -25,34 +47,44 @@
     sslcontext = ssl.create_default_context(cafile="cat-aii-root.cer")
 
 
-async def confidence_indicator(request):
-    """ USPS AII API view """
+@app.post(
+    "/person",
```

Comment:
Path should be `confidence_indicator` to more closely model the downstream USPS api. Currently we have Kong using `<give-url>/usps/` sending all requests to that url to the /confidence_indicator of usps, but in order to support additional usps endpoints, we should be mimicking their api.